core/src/components/picker-column-option/picker-column-option.tsx

```diff
@@ -0,0 +1,55 @@
+import type { ComponentInterface } from '@stencil/core';
+import { Component, Element, Host, Prop, h } from '@stencil/core';
+import type { Attributes } from '@utils/helpers';
+import { inheritAriaAttributes } from '@utils/helpers';
+
+import { getIonMode } from '../../global/ionic-global';
+
+@Component({
+  tag: 'ion-picker-column-option',
+  shadow: true,
+})
+export class PickerColumnOption implements ComponentInterface {
+  private optionId = `ion-picker-opt-${pickerOptionIds++}`;
+
+  private inheritedAttributes: Attributes = {};
+
+  @Element() el!: HTMLElement;
+
+  /**
+   * If `true`, the user cannot interact with the select option.
+   */
+  @Prop() disabled = false;
+
+  /**
+   * The text value of the option.
+   */
+  @Prop() value?: any | null;
+
+  componentWillLoad() {
+    this.inheritedAttributes = inheritAriaAttributes(this.el);
+  }
+
+  render() {
+    const { value, disabled, inheritedAttributes } = this;
+    const ariaLabel = inheritedAttributes['aria-label'] || null;
+
+    return (
+      <Host id={this.optionId} class={getIonMode(this)}>
+        <button
+          tabindex="-1"
+          aria-label={ariaLabel}
+          class={{
+            'picker-opt': true,
+            'picker-opt-disabled': !!disabled,
```

Comment:
We may want to use `picker-item`/`picker-item-disabled` instead to match the existing implementation: https://github.com/ionic-team/ionic-framework/blob/4f050ad437e4087b05edba8686691f61c9cd70c2/core/src/components/picker-column/picker-column.tsx#L470 This would make it easier to copy styles over later.

Reply:
Oh, nevermind, I see you were copying this: https://github.com/ionic-team/ionic-framework/blob/4f050ad437e4087b05edba8686691f61c9cd70c2/core/src/components/picker-legacy-column/picker-column.tsx#L420 I think I just got confused on which implementation is which 😆